packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -53,31 +55,33 @@ void main() {
                   onPressed: () { },
                 ),
                 floatingActionButtonLocation: location,
-                bottomNavigationBar: BottomAppBar(
+                bottomNavigationBar: const BottomAppBar(
                   shape: AutomaticNotchedShape(
-                    BeveledRectangleBorder(borderRadius: BorderRadius.circular(50.0)),
-                    ContinuousRectangleBorder(borderRadius: BorderRadius.circular(30.0)),
+                    ContinuousRectangleBorder(borderRadius: 30.0),
                   ),
                   notchMargin: 10.0,
                   color: Colors.green,
-                  child: const SizedBox(height: 100.0),
+                  child: SizedBox(height: 100.0),
                 ),
               ),
             ),
           ),
         ),
       );
     }
+
     await pump(FloatingActionButtonLocation.endDocked);
     await expectLater(
       find.byKey(key),
       matchesGoldenFile('bottom_app_bar.custom_shape.1.png'),
+      skip: !Platform.isLinux,
```

Comment:
I think we're reducing the test scope here. Split it into multiple tests if needed.

Reply:
I thought golden tests were only supposed to be run on linux? Or am I mistaken?